src/splunk-events.ts

```diff
@@ -298,62 +362,102 @@ export default class SplunkEvents {
     })
       .then(() => {
         if (this.debug) {
-          console.log(
-            `${this.pendingEvents.length} events successfuly sent to splunk`
-          )
+          console.log(`${events.length} events successfuly sent to splunk`)
         }
+      })
+      .catch((e) => {
+        if (this.debug) {
+          console.warn('Error sending events to splunk.', e)
+        }
+
+        throw e
+      })
+  }
+
+  private _backoffFlush = () => {
+    if (this.isBackoffInProgress) {
+      return
+    }
+
+    this.isBackoffInProgress = true
+
+    const backoffMultiplier = 2
+
+    const executeFlush = (depth = 0) => {
+      return this.flush()
+        .then(() => {
+          this.events = []
+          this.isBackoffInProgress = false
+        })
+        .catch(() => {
+          const waitTime = backoffMultiplier ** depth * 1_000
```

Comment:
Just to make it clearer. Wdyt?
```suggestion
          const waitTime = (backoffMultiplier ** depth) * 1_000
```

Reply:
I think eslint will remove the parenthesis (because of [this rule](https://eslint.org/docs/rules/no-extra-parens)) :-/

Reply:
Aw ok